client/cl_property.lua

```diff
@@ -160,8 +160,14 @@ end
 function Property:RegisterGarageZone()
     if not next(self.propertyData.garage_data) then return end
 
-    if not (self.has_access or self.owner) or not self.owner then
-        return
+    if Config.AllowAccessToGarage then
+        if not (self.has_access or self.owner) then    
+            return
+        end 
+    else
+        if not (self.has_access or self.owner) or not self.owner then
+            return
+        end
```

Comment:
Can we simplify this down to just 

`if not self.owner or not (self.has_access and Config.AllowAccessToGarage) return end`
I think this is the equivalent, you'd have to double check me I didnt check all the cases with this :P 

And maybe add a comment too
If player is not the owner or they dont have access, or they do have access but its configured to only allow owner access